src/docs/development/platform-integration/web.md

```diff
@@ -9,32 +9,36 @@ As of the 1.12 release, web support for Flutter
 is available on the beta channel.
 **We don't recommend deploying a web app to production.**
 
+### What sort of apps are not ideal for Flutter on the web?
```

Comment:
Can we use the language below for consistency with the /web page? 

> Not every HTML scenario is ideally suited for Flutter at this time. For example, text-rich flow-based content such as blog articles benefit from the document-centric model that the web is built around, rather than the app-centric services that a UI framework like Flutter can deliver.

Reply:
+1 this as an answer sounds good to me